Dockerfile.base

```diff
@@ -1,6 +1,9 @@
-FROM node:12.18.2-alpine AS frontend
+FROM node:16.13.0 AS frontend
```

Comment:
Did you check this? If this change is needed..

Reply:
needed as discussed

Reply:
`universe-utils` requires node version `>16`